modules/argocd/argocd-project/helm/values.yaml

```diff
@@ -0,0 +1,46 @@
+name: "system"
+
+namespace: "argocd"
+
+description: "Example Project"
+
+clusterResourceWhitelist:
+  - group: "*"
+    kind: "*"
+namespaceResourceBlacklist:
+#  - group: ''
+#    kind: ResourceQuota
+#  - group: ''
+#    kind: LimitRange
+#  - group: ''
+#    kind: NetworkPolicy
+namespaceResourceWhitelist:
+#  - group: 'apps'
+#    kind: Deployment
+#  - group: 'apps'
+#    kind: StatefulSet
+roles:
+#  - name: read-only
+#    description: Read-only privileges to my-project
+#    policies:
+#      - p, proj:my-project:read-only, applications, get, my-project/*, allow
+#    groups:
+#      - my-oidc-group
+# Destination cluster.
+destinations:
+  - namespace: "*"
+    server: "https://kubernetes.default.svc"
```

Comment:
```suggestion
    server: "https://kubernetes.default.svc"
sourceNamespaces: []
```

Reply:
I don't think we need `sourceNamespaces: []`. actually, `destinations:` is `destinations: []` and we can add more namespace and server in the `destinations`. please take a look at this example - https://github.com/argoproj/argo-cd/blob/master/docs/operator-manual/project.yaml . I'm referencing it.

Reply:
here is my example - https://github.com/aws-ia/terraform-aws-eks-blueprints/pull/1413/files#diff-35e3176ae40868f713cd61ae43525df41808209c100b21562ffaf66e699bc42d

Reply:
Hi @dtherhtun 
destinations are different sourceNamespaces
Here are the docs https://argo-cd.readthedocs.io/en/stable/operator-manual/app-any-namespace/#managing-applications-in-other-namespaces

This is used when you have ArgoCD applications in different namespaces other than `argocd` namespace that the application controller is installed.
The use case is when a single ArgoCD instance is managing different remote clusters, and you want to create namespaces for tenants to install ArgoCD Applications in different namespaces with the same name (ie addon name), but for different clusters.
See my example here:
- https://github.com/csantanapr/terraform-aws-eks-blueprints/blob/argo-multi-cluster/examples/gitops/argocd-multi-cluster/hub-cluster/helm-argocd/values.yaml#L85
- https://github.com/csantanapr/terraform-aws-eks-blueprints/blob/argo-multi-cluster/examples/gitops/argocd-multi-cluster/spoke-cluster-template/main.tf#L353

Reply:
@csantanapr updated and thanks.